server/README.md

```diff
@@ -0,0 +1,34 @@
+# Wflow ZMQ Server
+Call [Wflow](https://github.com/Deltares/Wflow.jl) functions exposed through the Basic Model
+Interface (BMI) implementation of Wflow and three additonal Wflow functions related to model
+states and start time of the model in Unix time, using [ZeroMQ](https://zeromq.org/) with
+TCP data transport. The Wflow ZMQ Server allows users to interact with a Wflow model from
+many other programming languages with ZeroMQ bindings. An example is the use of
+[OpenDa](https://openda.org/) Java software for data-assimilation. The coupling of OpenDA
+and Wflow through a black box model approach is too slow because it requires a restart of
+Wflow (multiple times), while this is not required with a client-server approach.  
+
+Specify the `WflowServer` environment on startup, in the current directory:
+```
+julia --project=.
+```
+and start the Wflow ZMQ Server, listening on port `<port>` (default: 5555) as follows:
+```julia-repl
+julia> using WflowServer
+julia> WflowServer.start() <port>
```

Comment:
Perhaps providing a one-liner would be helpful here for those not wanting to start an interactive julia session.

Something like `julia --project=. --eval='using WflowServer; WflowServer.start(5555)'

Reply:
Thanks for providing this one-liner, I have updated the README and listed the different ways how the Wflow ZMQ Serer can be started.